src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -111,3 +113,108 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='JSON encoded parameters configuration. Use @{file} to load from a file. '
                         'For example: [{"isAdmin":true, "certificateCommonName": "test.com", '
                         '"certificateIssuerThumbprint": "22B4AE296B504E512DF880A77A2CAE20200FF922"}]')
+
+    # application-type
+    with self.argument_context('sf application-type') as c:
+        c.argument('application_type_name', options_list=['--name', '--application-type-name'], help='Specify the application type name.')
+
+    # application-type-version
+    with self.argument_context('sf application-type-version') as c:
+        c.argument('application_type_name', options_list=['--name', '--application-type-name'], help='Specify the application type name.')
+
+    with self.argument_context('sf application-type-version create') as c:
+        c.argument('version', options_list=['--version', '--application-type-version'], help='Specify the application type version.')
+        c.argument('package_url', help='Specify the url of the application package sfpkg file.')
+
+    # application
+    with self.argument_context('sf application') as c:
+        c.argument('application_name', options_list=['--name', '--application-name'], help='Specify the application name.')
+
+    with self.argument_context('sf application update', validator=validate_update_application) as c:
+        c.argument('application_type_version', options_list=['--version', '--application-type-version'], help='Specify the application type version.')
+        c.argument('application_parameters', options_list=['--parameters', '--application-parameters'], action=addAppParamsAction, nargs='+',
+                   help='Specify the application parameters as key/value pairs. These parameters must exist in the application manifest.'
+                   'for example: --application-parameters param1=value1 param2=value2')
+        c.argument('minimum_nodes', options_list=['--min-nodes', '--minimum-nodes'],
+                   help='Specify the minimum number of nodes where Service Fabric will reserve capacity for this application.')
+        c.argument('maximum_nodes', options_list=['--max-nodes', '--maximum-nodes'],
+                   help='Specify the maximum number of nodes on which to place an application.')
```

Comment:
Do we have default value or range limitation for the two parameters?

Reply:
yes, added the limits in the description and validation logic